ledger-test-tool/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_15/ExplicitDisclosureIT.scala

```diff
@@ -378,7 +265,7 @@ final class ExplicitDisclosureIT extends LedgerTestSuite {
 
   test(
     "EDInconsistentDisclosedContracts",
-    "The ledger rejects disclosed contracts with inconsistent metadata",
+    "The ledger rejects disclosed contracts with invalid disclosed contract",
```

Comment:
I think we need a case when the disclosed contract payload is not populated on a disclosed-contract record

Reply:
I omitted since atm missing `create_event_payload` defaults to the deprecated format which we plan to drop soon. But I guess I can add it now as well since we're not concerned with backwards compatibility of snapshot versions.